Weatherman.py

```diff
@@ -0,0 +1,247 @@
+        file_size = reading_file(file_name)
+        dateM_list = []
+        maxiM_temperature_list = []
+        make_tlist = []
+        miniM_temperature_list = []
+        meanH_list_humidity = []
+        file_contents = []
+        file_contents = f.readline()
+        i = 1
+        while i < file_size:
+            file_contents = f.readline()
+            make_tlist = file_contents.split(',')
+            maxiM_temperature_list.append(make_tlist[1])
+            miniM_temperature_list.append(make_tlist[3])
+            meanH_list_humidity.append(make_tlist[9])
+            dateM_list.append(make_tlist[0])
+            i += 1
+        maxiM_average = average_temp(maxiM_temperature_list)
+        minM_average = average_temp(miniM_temperature_list)
+        humidity_average = average_temp(meanH_list_humidity)
+        print(f'Average Highest Temperature : {maxiM_average}C')
+        print(f'Average Lowest Temperature : {minM_average}C')
+        print(f'Average Humidity : {humidity_average}%')
+        bar_charts(maxiM_temperature_list, miniM_temperature_list)
+        horizontal_bar_charts(maxiM_temperature_list, miniM_temperature_list)
+
+
+def average_temp(l1):
+    sum1 = 0
+    count = 0
+    i = 0
+    while i < len(l1):
+        if l1[i] != '':
+            sum1 = sum1 + float(l1[i])
+            count += 1
+        i += 1
+    return sum1 / count
+
+
+def horizontal_bar_charts(l1, l2):
+    temp1 = 0
+    Red = "\033[31m"
+    Blue = "\033[34m"
+    White = "\033[37m"
+    print(White + 'The Horizontal Bar charts for temperatures are :')
+    while temp1 < len(l2):
+        if l2[temp1] == '':
+            print('', end='')
+        else:
+            temp2 = 0
+            temp3 = 0
+            temp4 = 0
+            print(temp1 + 1, end='')
+            while temp2 < float(l2[temp1]):
+                print(Blue + '+', end='')
+                temp2 += 1
+            while temp3 <= temp1:
+                if l1[temp1] == '':
+                    print('', end='')
+                else:
+                    while temp4 < float(l1[temp1]):
+                        print(Red + '+', end='')
+                        temp4 += 1
+                temp3 += 1
+            print('')
+        temp1 += 1
+    print('')
+
+
+def bar_charts(l1, l2):
+    k = 0
+    Red = "\033[31m"
+    Blue = "\033[34m"
+    print('The bar chart for temperatures are :')
+    while k < len(l1):
+        if l1[k] == '':
+            print('', end='')
+        else:
+            t = 0
+            m = 0
+            n = 0
+            while t < float(l1[k]):
+                print(Red + '+', end='')
+                t += 1
+            print('')
+            while m <= t:
+                if l2[t] == '':
+                    print('', end='')
+                else:
+                    while n < float(l2[t]):
+                        print(Blue + '+', end='')
+                        n += 1
+                m += 1
+            print('')
+        k += 1
```

Comment:
this all code not readable and understandable make sure your code look well and read well